hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -229,119 +229,243 @@ def hpat_pandas_series_not_equal_impl(lhs, rhs):
 
 @overload_method(SeriesType, 'add')
 def hpat_pandas_series_add(lhs, rhs):
-    '''
+    """
     Pandas Series operator 'add' implementation.
         https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.add.html#pandas.Series.add
 
     Algorithm: result = lhs.add(other, level=None, fill_value=None, axis=0)
 
     Where:
                lhs: pandas.series
-             other: pandas.series
+             other: pandas.series or scalar value
              level: unsupported
         fill_value: unsupported
               axis: unsupported
             result: pandas.series result of the operation
 
     Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op5
-    '''
+    """
 
     _func_name = 'Operator add().'
 
-    if not isinstance(lhs, SeriesType) or not isinstance(rhs, SeriesType):
-        raise TypingError('{} The object must be a pandas.series. Given lhs: {}, rhs: {}'.format(_func_name, lhs, rhs))
+    if not isinstance(lhs, SeriesType):
+        raise TypingError('{} The object must be a pandas.series. Given lhs: {}'.format(_func_name, lhs))
+
+    if isinstance(rhs, SeriesType):
+        def hpat_pandas_series_add_impl(lhs, rhs):
+            """
+            Test:  python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_op5
+            """
+
+            return pandas.Series(lhs._data + rhs._data)
+
+        return hpat_pandas_series_add_impl
+
+    if isinstance(rhs, types.Integer) or isinstance(rhs, types.Float):
```

Comment:
I'm afraid it covers `A + 5` only. I think we will have other expressions like `5 + A`

Reply:
Please treat my previous comment as "no sense" due to this is a function `series.add(5)` but not an operator `series + 5.0`
Sorry.
This PR is completed and going to be merged.